src/main/java/uk/gov/pay/directdebit/payments/api/CollectPaymentResponse.java

```diff
@@ -26,6 +28,12 @@
     @JsonProperty
     private Long amount;
 
+    @JsonProperty("mandate_id")
+    private String mandateId;
```

Comment:
If this is the mandate external Id then there is a `MandateExternalId` class to represent it.